ant/apple/installer.xml

```diff
@@ -199,6 +159,45 @@
         </copy>
 
         <property name="build.sandboxed" value="true"/>
+        <antcall target="copy-bundle">
+            <param name="build.copydir" value="${dist.dir}/${project.name}.app"/>
+        </antcall>
+
+        <!-- Disable signing and append "-community" if id is missing -->
+        <exec executable="bash" failonerror="false" resultproperty="codesign.qz">
+            <arg value="-c"/>
+            <arg value="security find-identity -v |grep '(${apple.packager.signid})'"/>
+        </exec>
+
+        <condition property="codesign.qz.suffix" value="" else="-community">
+            <equals arg1="${codesign.qz}" arg2="0"/>
+        </condition>
+
+        <exec executable="appdmg" failonerror="true">
+            <arg value="${build.dir}/appdmg.json"/>
+            <arg value="${out.dir}/${project.filename}${build.type}-${build.version}${codesign.qz.suffix}.dmg"/>
+        </exec>
+    </target>
+
+    <target name="codesign-jre" if="codesign.mac" unless="jre.skip">
+        <property file="ant/apple/apple.properties"/>
+    </target>
+
+    <target name="copy-bundle">
```

Comment:
What was wrong with the old target name `build-app`?  I think it's more descriptive.

Reply:
https://github.com/qzind/tray/blob/e2b0f5a4a2aea510b6c502898f40c4aca161e592/ant/apple/installer.xml#L20 it is stiill there, git diff is a bit messy

Reply:
The commit message says `remove redundant code`, yet this block is still redundant.

```xml
        <!-- set payload files executable -->
        <chmod perm="a+x" type="file">
            <fileset dir="${build.copydir}">
                <include name="**/${project.name}"/>
                <include name="**/Resources/uninstall"/>
                <include name="**/bin/*"/>
                <include name="**/lib/jspawnhelper"/>
            </fileset>
        </chmod>
```

Reply:
missed that somehow. it is not needed.